integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/common/Utils.java

```diff
@@ -14,4 +19,20 @@ public static String getSystemProperty(String parameters) {
     public static String generateId(String prefix) {
         return prefix + "-" + UUID.randomUUID().toString().substring(0, 4);
     }
+
+    public static Optional<String> getEndpointBaseUrl(String endpoint) {
+        Matcher matcher = ENDPOINT_URL_REGEX.matcher(endpoint);
+        if (matcher.find()) {
+            return Optional.of(matcher.group(1));
+        }
+        return Optional.empty();
+    }
+
+    public static Optional<String> getEndpointPathUrl(String endpoint) {
+        Matcher matcher = ENDPOINT_URL_REGEX.matcher(endpoint);
+        if (matcher.find()) {
+            return Optional.of(matcher.group(2));
+        }
+        return Optional.empty();
+    }
```

Comment:
Can this be placed in some dedicated class? I would rather avoid using generic classes like `Utils`.

Reply:
yes, makes sense.
Done